src/pages/studyView/TableUtils.tsx

```diff
@@ -47,27 +87,49 @@ export function getCancerGeneFilterToggleIcon(isFilteredByCancerGeneList:boolean
     return <span data-test='cancer-gene-filter' className={classnames(styles.cancerGeneIcon, styles.displayFlex)} style={{color: isFilteredByCancerGeneList ? ICON_FILTER_ON : ICON_FILTER_OFF}}><i className='fa fa-filter'></i></span>;
 }
 
-export function getFreqColumnRender(type: 'mutation' | 'cna', numberOfSamplesProfiled: number, numberOfAlteredCases: number, matchingGenePanelIds: string[], toggleModal: (panelName: string) => void, style?:CSSProperties) {
+type FreqColumnType = 'mutation' | 'fusion' | 'cna';
+
+export function getFreqColumnRender(type: FreqColumnType, numberOfSamplesProfiled: number, numberOfAlteredCases: number, matchingGenePanelIds: string[], toggleModal?: (panelName: string) => void, style?: CSSProperties) {
+    const detailedTypeInfo = type === 'mutation' ? 'mutations' : (type === 'cna' ? 'copy number alterations' : 'fusions')
     const addTotalProfiledOverlay = () => (
         <span style={{display: 'flex', flexDirection: 'column'}} data-test='freq-cell-tooltip'>
-            <span>{`# of samples profiled for ${type === 'mutation' ? 'mutations' : 'copy number alterations'} in this gene: ${numberOfSamplesProfiled.toLocaleString()}`}</span>
+            <span>{`# of samples profiled for ${detailedTypeInfo} in this gene: ${numberOfSamplesProfiled.toLocaleString()}`}</span>
             <GenePanelList
                 genePanelIds={matchingGenePanelIds}
-                toggleModal={toggleModal}
+                toggleModal={toggleModal!}
             />
         </span>
     );
+
+    function getCellContent() {
```

Comment:
kinda strange pattern here.   uses style variable from closure.  i would rather just see this inside of the markup as it was